histomicstk/utils/sample_pixels.py

```diff
@@ -149,7 +161,16 @@ def _sample_pixels_tile(slide_path, iter_args, positions, sample_fraction,
             continue
 
         # get current tile image
-        im_tile = tile['tile'][:, :, :3]
+        # TODO - check if single channel
```

Comment:
We'll want to remove the `TODO` part of these comments since they are done.

Reply:
All the TODO's and some print functions used for debugging are removed from the code.